db/migrate/20151018090641_create_persistent_volume_claim.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePersistentVolumeClaim < ActiveRecord::Migration
+  def change
+    create_table :persistent_volume_claims do |t|
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :persistent_volume, :type => :bigint
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.timestamp  :created_on
+      t.string     :resource_version
+      t.text       :desired_access_modes, :array => true, :default => []
+      t.string     :volume_name
+      t.string     :phase
+      t.text       :actual_access_modes, :array => true, :default => []
+      t.text       :capacity
```

Comment:
Technically the `capacity` and the `access_modes` can differ because it's up to kubernetes to bound a Persistent Volume and a Persistent Volume Claim. Currently the algorithm that does the matching is looking for equality (so technically they would be the same), but in theory it's also possible a less strict matching (E.g. if you ask for a 5Gb PV you can get a 10Gb one as well, if you're asking for RWO you could get a RWX).